torch/testing/_internal/opinfo/definitions/nested.py

```diff
@@ -180,6 +180,23 @@ def sample_inputs_njt_reduction(
                     njt, kwargs={**op_kwargs, "dim": dim, "keepdim": keepdim}
                 )
 
+        for dim in range(njt._ragged_idx + 1, njt.dim() - 1):
+            for keepdim in [False, True]:
+                yield SampleInput(
+                    njt, kwargs={**op_kwargs, "dim": (dim, dim + 1), "keepdim": keepdim}
+                )
+
+        for dim in range(njt._ragged_idx + 1, njt.dim() - 2):
+            for keepdim in [False, True]:
+                yield SampleInput(
+                    njt,
+                    kwargs={
+                        **op_kwargs,
+                        "dim": (dim, dim + 1, dim + 2),
+                        "keepdim": keepdim,
+                    },
+                )
+
```

Comment:
Do we actually have any reductions that support `keepdim=False` right now? Maybe better to just update this to only run with keepdim=True so that we get the coverage and add a todo to reenable keepdim when we start supporting that case.